jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/FormField.java

```diff
@@ -0,0 +1,322 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.*;
+import com.epam.jdi.light.angular.elements.composite.*;
+import com.epam.jdi.light.common.*;
+import com.epam.jdi.light.elements.base.*;
+import com.epam.jdi.light.elements.common.*;
+import com.epam.jdi.light.elements.complex.*;
+import com.epam.jdi.light.logger.*;
+import com.epam.jdi.tools.map.*;
+import org.openqa.selenium.*;
+
+import java.util.*;
+
+import static com.epam.jdi.light.common.TextTypes.*;
+import static com.epam.jdi.light.driver.WebDriverFactory.*;
+
+/**
+ To see an example of FormField web element please visit https://material.angular.io/components/form-field/overview#form-field-appearance-variants
+ **/
+
+public class FormField extends UIBaseElement<FormFieldsAssert> {
+
+    private String matFormFieldPrefix = "//mat-form-field[@";
+    private String autocompleteAttrPrefix = "_ngcontent-";
+
+
+    private String containerAttribute;
+
+    private String getContainerAttribute() {
+        final String[] curAttr = {""};
+        MapArray<String, String> attributesAndValues = this.core().getAllAttributes();
+        List<String> attributes = attributesAndValues.keys();
+        attributes.forEach(attr -> {
+            boolean doesContain = attr.contains(autocompleteAttrPrefix);
+            if (doesContain) {
+                curAttr[0] = attr;
+            }
+        });
+        return curAttr[0];
+    }
+
+    public UIElement getContainer() {
```

Comment:
This method is only used in the current class. Maybe the modifier should be protected.

Reply:
Done.